org.omg.kerml.xpect.tests/library/BaseFunctions.kerml

```diff
@@ -32,7 +32,7 @@ standard library package BaseFunctions {
 	abstract function '['{ in x: Anything[0..*] nonunique; in y: Anything[0..*] nonunique; 
 		return : Anything[0..*] nonunique;
 	}
-	abstract function '#'{ in seq: Anything[0..*] ordered nonunique; in index: Anything[0..*] ordered nonunique; 
+	abstract function '#'{ in seq: Anything[0..*] ordered nonunique; in index: Positive[1..*] ordered nonunique; 
 		return : Anything[0..1];
```

Comment:
In some languages (e.g. Python) negative indices are allowed to index from the tailend of a sequence. Very useful, but probably something for an RTF.

Reply:
Note that this is only a change in the library model file used for Xpect testing, to update it to the latest normative version. The normative library model from the beta specification already had the type here as `Positive`. So changing that would be a separate issue.